tests/test_integration.py

```diff
@@ -61,7 +61,12 @@ def assign_status_finished():
     # check if visualisation files are stored
     def microreact_status_finished():
         status = client.get("/status/" + p_hash)
-        assert read_data(status)['microreact'] == 'finished'
+        microreact_clusters_status = read_data(status)["microreactClusters"]
+        assert len(microreact_clusters_status) > 0
+        assert all(
+            status == "finished"
```

Comment:
It would be nice to also check that you get microreact statuses back before the whole microreact run is finished, for expected clusters. But that might be flaky.

Reply:
good idea..yeah might be hard since it polls in test